src/main/play-doc/operation/Install.md

```diff
@@ -30,6 +30,12 @@ On Ubuntu 14.10 and newer, you can use the built in package manager.
 sudo apt-get install openjdk-8-jdk
 ```
 
+Note, newly launched instances may require a repository update, for example on Ubuntu and Debian run:
+
+```bash
+sudo apt-get update
+```
+
```

Comment:
I think just combine this with the command on L30 and leave the note off, e.g. `sudo apt-get update && sudo apt-get install openjdk-8-jdk`.